include/z64scene.h

```diff
@@ -534,119 +534,119 @@ typedef union {
+    /* 0x0C */ SCENE_HAKASHITA, // Beneath the Graveyard
+    /* 0x0D */ SCENE_AYASHIISHOP, // Curiosity Shop
     /* 0x0E */ SCENE_UNSET_E,
     /* 0x0F */ SCENE_UNSET_F,
-    /* 0x10 */ SCENE_OMOYA,
-    /* 0x11 */ SCENE_BOWLING,
-    /* 0x12 */ SCENE_SONCHONOIE,
-    /* 0x13 */ SCENE_IKANA,
-    /* 0x14 */ SCENE_KAIZOKU,
-    /* 0x15 */ SCENE_MILK_BAR,
-    /* 0x16 */ SCENE_INISIE_N,
-    /* 0x17 */ SCENE_TAKARAYA,
-    /* 0x18 */ SCENE_INISIE_R,
-    /* 0x19 */ SCENE_OKUJOU,
-    /* 0x1A */ SCENE_OPENINGDAN,
-    /* 0x1B */ SCENE_MITURIN,
-    /* 0x1C */ SCENE_13HUBUKINOMITI,
-    /* 0x1D */ SCENE_CASTLE,
-    /* 0x1E */ SCENE_DEKUTES,
-    /* 0x1F */ SCENE_MITURIN_BS,
-    /* 0x20 */ SCENE_SYATEKI_MIZU,
-    /* 0x21 */ SCENE_HAKUGIN,
-    /* 0x22 */ SCENE_ROMANYMAE,
-    /* 0x23 */ SCENE_PIRATE,
-    /* 0x24 */ SCENE_SYATEKI_MORI,
-    /* 0x25 */ SCENE_SINKAI,
-    /* 0x26 */ SCENE_YOUSEI_IZUMI,
-    /* 0x27 */ SCENE_KINSTA1,
-    /* 0x28 */ SCENE_KINDAN2,
-    /* 0x29 */ SCENE_TENMON_DAI,
-    /* 0x2A */ SCENE_LAST_DEKU,
-    /* 0x2B */ SCENE_22DEKUCITY,
-    /* 0x2C */ SCENE_KAJIYA,
-    /* 0x2D */ SCENE_00KEIKOKU,
-    /* 0x2E */ SCENE_POSTHOUSE,
-    /* 0x2F */ SCENE_LABO,
-    /* 0x30 */ SCENE_DANPEI2TEST,
+    /* 0x10 */ SCENE_OMOYA, // Mama's House & Barn
+    /* 0x11 */ SCENE_BOWLING, // Honey & Darling's Shop
+    /* 0x12 */ SCENE_SONCHONOIE, // The Mayor's Residence
+    /* 0x13 */ SCENE_IKANA, // Ikana Canyon
+    /* 0x14 */ SCENE_KAIZOKU, // Pirates' Fortress
+    /* 0x15 */ SCENE_MILK_BAR, // Milk Bar
+    /* 0x16 */ SCENE_INISIE_N, // Stone Tower Temple
+    /* 0x17 */ SCENE_TAKARAYA, // Treasure Chest Shop
+    /* 0x18 */ SCENE_INISIE_R, // Inverted Stone Tower Temple
+    /* 0x19 */ SCENE_OKUJOU, // Clock Tower Rooftop
+    /* 0x1A */ SCENE_OPENINGDAN, // Before Clock Town
+    /* 0x1B */ SCENE_MITURIN, // Woodfall Temple
+    /* 0x1C */ SCENE_13HUBUKINOMITI, // Path to Mountain Village
+    /* 0x1D */ SCENE_CASTLE, // Ancient Castle of Ikana
+    /* 0x1E */ SCENE_DEKUTES, // Deku Scrub Playground
+    /* 0x1F */ SCENE_MITURIN_BS, // Odolwa's Lair
+    /* 0x20 */ SCENE_SYATEKI_MIZU, // Town Shooting Gallery
+    /* 0x21 */ SCENE_HAKUGIN, // Snowhead Temple
+    /* 0x22 */ SCENE_ROMANYMAE, // Milk Road
+    /* 0x23 */ SCENE_PIRATE, // Pirates' Fortress Interior
+    /* 0x24 */ SCENE_SYATEKI_MORI, // Swamp Shooting Gallery
+    /* 0x25 */ SCENE_SINKAI, // Pinnacle Rock
+    /* 0x26 */ SCENE_YOUSEI_IZUMI, // Fairy's Fountain
+    /* 0x27 */ SCENE_KINSTA1, // Swamp Spider House
+    /* 0x28 */ SCENE_KINDAN2, // Oceanside Spider House
+    /* 0x29 */ SCENE_TENMON_DAI, // Astral Observatory
+    /* 0x2A */ SCENE_LAST_DEKU, // Deku Trial
+    /* 0x2B */ SCENE_22DEKUCITY, // Deku Palace
+    /* 0x2C */ SCENE_KAJIYA, // Mountain Smithy
+    /* 0x2D */ SCENE_00KEIKOKU, // Termina Field
+    /* 0x2E */ SCENE_POSTHOUSE, // Post Office
+    /* 0x2F */ SCENE_LABO, // Marine Research Lab
+    /* 0x30 */ SCENE_DANPEI2TEST, // Dampe's House
```

Comment:
```suggestion
    /* 0x30 */ SCENE_DANPEI2TEST, // Dampe's House and Day Three Grave
```
I've never been fond of calling this scene his house, because yes there is a room which is his house, with a bed and stuff, but most players know this scene for the Dampe digging minigame and Bigpo fight and bottle, and the house is a minor part of it. Not convinced `Day Three Grave` is the best but the best suggestion I got to keep it brief.

Reply:
Well, the game calls it "Beneath the Graveyard", which is itself kind of confusing because it's a completely different scene than the *other* "Beneath the Graveyard":
![image](https://user-images.githubusercontent.com/12245827/188239292-6caf1227-47aa-4526-b727-a9cdf4b146f0.png)

I'll try to incorporate them both into the comment.